lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class Order
+  
+  attr_reader :orders, :menu
+  
+  def initialize
+    @orders = []
+  end
+
+  def order_confirmation
+    delivery_time
+    puts "Thanks for your order! It will be delivered in 45 minutes at #{@time.strftime("%H:%M")}."
+    send_sms
+  end
+
+  def delivery_time
+    @time = Time.new + 45 * 60
+  end
+  
+  def send_sms
+    account_sid = ENV["TWILIO_ACCOUNT_SID"]
+    auth_token = ENV["TWILIO_AUTH_TOKEN"]
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    message = @client.messages.create(
+      body: "Thanks for your order! It will be delivered in 45 minutes at #{@time.strftime("%H:%M")}.",
+      to: ENV["MY_PHONE_NUMBER"],
+      from: ENV["TWILIO_MAGIC_NUMBER"])
+  end
```

Comment:
Very nice. Maybe, having delivery_time return the time string would keep the time responsibility away from this method.
Like:
```
 def delivery_time
    time = Time.new + 45 * 60
    time.strftime("%H:%M")
  end
```

Reply:
Good idea. Would make it easier passing the time between methods